README.md

```diff
@@ -45,6 +45,64 @@ To use the Alchemy provider plugin in most commands, set it via the `--network`
 ape console --network ethereum:goerli:alchemy
 ```
 
+To connect to Alchemy from a Python script, use the `networks` top-level manager:
+
+```python
+from ape import networks
+
+with networks.parse_network_choice("ethereum:mainnet:alchemy") as provider:
+    ...
+```
+
+### Transaction Traces
```

Comment:
I think you need to update main (including your remote's main with the upstream main) to get some of these diffs out of the way since these are not part of the PR

Reply:
I see. Just merged main to get the commits I've missed